cli/integration_tests/basic_monorepo/run_summary/enable.t

```diff
@@ -0,0 +1,70 @@
+Setup
+  $ . ${TESTDIR}/../../setup.sh
+  $ . ${TESTDIR}/../setup.sh $(pwd)
+
+# Delete all run summaries to start
+  $ rm -rf .turbo/runs
+
+# Tests
+| env var | flag    | summary? |
+| ------- | ------- | -------- |
+| true    | missing | yes      |
+| true    | true    | yes      |
+| true    | false   | no       |
+
+| false   | missing | no       |
+| false   | true    | yes      |
+| false   | false   | no       |
+
+| missing | missing | no       |
+| missing | true    | yes      |
+| missing | false   | no       |
```

Comment:
I'm not sure for sure that this is the right result for each combination, but it's a starting point

Reply:
All of these outcomes looks solid to me.